docs/my-website/docs/proxy/ui.md

```diff
@@ -124,6 +124,7 @@ GENERIC_CLIENT_SECRET = "G*******"
 GENERIC_AUTHORIZATION_ENDPOINT = "http://localhost:9090/auth"
 GENERIC_TOKEN_ENDPOINT = "http://localhost:9090/token"
 GENERIC_USERINFO_ENDPOINT = "http://localhost:9090/me"
+GENERIC_SCOPE = "openid profile email"
```

Comment:
is this a required variable ?

If it's optional can you clarify this on the docs

Reply:
It is not required.
I have clarified the usage with : 21552620edbe881180abdaf39f6a6a8a25abeab7